README.md

```diff
@@ -201,6 +201,19 @@ Please review the [attributes/default.rb](https://github.com/DataDog/chef-datado
 
 For general information on the Datadog Agent 6, please refer to the [datadog-agent](https://github.com/DataDog/datadog-agent/) repo.
 
+#### Windows Agent v6 installation
+
+Starting with version `>= 6.11`, the Agent v6 must be installed with a chef-datadog
```

Comment:
here I'd mention **Windows** Agent v6 again, for extra clarity

Reply:
also, there is no `chef-datadog`  cookbook, that's just the name of the repo ;) The cookbook is named `datadog`